gnomad_qc/v3/utils.py

```diff
@@ -70,3 +73,28 @@ def rekey_new_reference(
         t = t.key_by("locus", "alleles").drop("locus_original")
 
     return t
+
+
+# TODO: Needs to be modified to fix the problem of the alt het correction on a split MT
+def hom_alt_depletion_fix(
+        mt: hl.MatrixTable,
+        af_expr: hl.expr.Float32Expression,
+        af_cutoff: float = 0.01,
+        ab_cutoff: float = 0.9,
+) -> hl.MatrixTable:
+    """
+    Adjust MT genotypes with temporary fix for the depletion of homozygous alternate genotypes.
+
+    :param mt: Input MT to add hom alt depletion genotype fix to
+    :param af_expr: Allele frequency expression to determine variants that need the hom alt fix
+    :param af_cutoff: Allele frequency cutoff for variants that need the hom alt fix
+    :param ab_cutoff: Allele balance cutoff to determine which genotypes need the hom alt fix
+    :return: MatrixTable with genotypes adjusted for the hom alt depletion fix
+    """
+    return mt.annotate_entries(
+        GT=hl.cond(
+            (af_expr > af_cutoff) & mt.GT.is_het() & ((mt.AD[1] / mt.DP) > ab_cutoff),
```

Comment:
Tim suggested putting the `mt.GT.is_het()` as the first filter here for efficiency

Reply:
the only other condition I added for the fix here was this `& ~mt.het_non_ref` (from adding this before splitting`mt = mt.annotate_entries(het_non_ref=mt.LGT.is_het_non_ref())`)